packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -303,56 +290,29 @@ export default class LocusInfo extends ChildEmitter {
   updateControls(controls) {
     if (controls && !isEqual(this.controls, controls)) {
       this.controls = controls;
-      console.info('LocusInfo:controls got updated');
     }
   }
 
   updateConversationUrl(conversationUrl) {
     if (conversationUrl && !isEqual(this.conversationUrl, conversationUrl)) {
       this.conversationUrl = conversationUrl;
-      this.emitChanges(EVENTS.CONVERSATION_URL_UPDATE, conversationUrl);
-      console.info('LocusInfo:conversationUrl got updated');
+      this.emitGlobal(EVENTS.CONVERSATION_URL_UPDATE, conversationUrl);
```

Comment:
The scope v. global concept is interesting.

Reply:
kudos to @arun3528 !